src/Formula/Parser.elm

```diff
@@ -56,6 +56,15 @@ parse =
     Parser.run (succeed identity |. spaces |= formula |. spaces |. end)
 
 
+
+{- Parse string to Substitution -}
```

Comment:
Improper format of exported function documentation. Should be:
```elm
{-| Parse a string into a Substitution.
-}
```
See also [<cite>Documentation Format</cite>](https://package.elm-lang.org/help/documentation-format).